packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -164,5 +162,21 @@ export default function useSiteEditorSettings() {
 		archiveLabels.archiveNameLabel,
 	] );
 
+	return defaultEditorSettings;
+}
+
+export default function useSiteEditorSettings() {
```

Comment:
I hope I can remove this function entirely but we can only do so when all of the site editor is relying on EditorProvider and/or the `BlockEditorProvider` usage within the site editor doesn't need the extra settings provided in the `EditorProvider`